Lib/distutils/_msvccompiler.py

```diff
@@ -262,6 +275,16 @@ def initialize(self, plat_name=None):
         ldflags_debug = [
             '/nologo', '/INCREMENTAL:NO', '/LTCG', '/DEBUG:FULL'
         ]
+        if self.use_clang_cl:
+            # Add target for clang
```

Comment:
I don't know if CPython makes use of any intrinsic or built-ins, but I had problems with both. 

Intrinsics required removing some of the include directories that are used with MSVC since clang doesn't need the `.h` files. 

Correct builtin support (on 32 bit Windows) required linking agsint the builtin .lib file.